README.md

```diff
@@ -30,13 +30,14 @@ Each pattern creates the following infrastructure on the VPC:
 
 - The virtual server (VSI) pattern deploys identical virtual servers across the VSI subnet tier in each VPC
 - The Red Hat OpenShift Kubernetes (ROKS) pattern deploys identical clusters across the VSI subnet tier in each VPC
+- The VPC pattern deploys networking infrastructure layer without any compute resources (no VSI, nor OpenShift cluster)
 - The mixed pattern provisions both of these elements
 
 For more information about the default configuration, see [Default Secure Landing Zone configuration](.docs/pattern-defaults.md).
 
-| Virtual server pattern           | Red Hat OpenShift pattern        | Mixed pattern                      |
-| -------------------------------- | -------------------------------- | ---------------------------------- |
-| ![VSI](./.docs/images/vsi.png)   | ![ROKS](./.docs/images/roks.png) | ![Mixed](./.docs/images/mixed.png) |
+| Virtual server pattern           | Red Hat OpenShift pattern        | VPC pattern                    | Mixed pattern                      |
```

Comment:
Seems like we would want to show the patterns in "order" from left to right with VPC first because it's the base. (At least from a deployable architectures POV.)
1. VPC
2. VSI
3. ROKs
4. Mixed